src/components/overlays/AddusersIDP/index.tsx

```diff
@@ -437,6 +439,21 @@ export const AddusersIDP = ({ id }: { id: string }) => {
         reader.onload = () => {
           if (!reader.result) return
           const content = reader.result.toString()
+          const CSVfileHeader = Object.keys(csv2json(content)[0])
+          if (
+            CSVfileHeader[0] !== 'companyUserId' ||
+            CSVfileHeader[1] !== 'firstName' ||
+            CSVfileHeader[2] !== 'lastName' ||
+            CSVfileHeader[3] !== 'email' ||
+            CSVfileHeader[4] !== 'identityProviders'
+          ) {
```

Comment:
we can write this smarter:
`const header = [ 'companyUserId', '...', '', '', '' ]
`if (!header.reduce((a,c,i) => a && (csvFileHeader[i] === c), true)) {